pages/glossary/programmatically-determined-context.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Programmatically determined context
+key: programmatically-determined-context
+---
+
+The programmatically determined context of a piece of information (_e.g._ an element) is all the additional information that has a [programmatically determinable](https://www.w3.org/TR/WCAG21/#dfn-programmatically-determinable) [relationship](https://www.w3.org/TR/WCAG21/#dfn-relationships) to it.
+
+It includes, but is not restricted to, `aria-label` attribute, element that is referred by `aria-labelledby` or `aria-describedby` attributes, ancestor `label` elements, `label` elements referring the current element with a `for` attribute, ancestor elements, table header associated with the cell containing the information, headings, text inside the same paragraph (`p` element), table cell, list item, …
```

Comment:
Editorial: Remove the ... (ellipsis).

Any thoughts on including the title attribute?

Reply:
I chose to not change it here given that @carlosapaduarte onboarded it with #555 and I didn't want to duplicate changes.